hil/ext/switches/dellnos9.py

```diff
@@ -128,18 +128,12 @@ def _get_vlans(self, interface):
         # worked reliably in the first place) and then find our interface there
         # which is not feasible.
 
-        response = self._get_port_info(interface).splitlines()
-
-        try:
-            index = response.index("Vlanmembership:") + 3
-        except ValueError:
-            # happens when our response has almost no info on a switchport
-            # because it's off. the response lacks the word "vlanmembership"
-            return []
-        if response[index] == '':
-            # a port with no tagged interface has no section with "T"
+        response = self._get_port_info(interface)
+        # finds a comma separated list of integers starting with "T"
+        match = re.search(re.compile(r'T(\d+)((,\d+)?)*'), response)
```

Comment:
You don't need the re.compile(), search will accept the regex string directly :)

Reply:
@geedo0 Done!